src/transformers/models/rt_detr/configuration_rt_detr.py

```diff
@@ -0,0 +1,343 @@
+# coding=utf-8
+# Copyright 2024 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" RT-DETR model configuration"""
+
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+from ..auto import CONFIG_MAPPING
+
+
+logger = logging.get_logger(__name__)
+
+RT_DETR_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "sbchoi/rtdetr_r50vd": "https://huggingface.co/sbchoi/rtdetr_r50vd/blob/main/config.json",
```

Comment:
To be updated (here and at other places) once the checkpoints are moved to the appropriate organization on the hub. I can help with that :)